pyseir/inference/model_fitter.py

```diff
@@ -697,6 +697,37 @@ def run_for_fips(cls, fips, n_retries=3, with_age_structure=False):
         return model_fitter
 
 
+def _execute_model_for_fips(fips):
+    if fips:
+        model_fitter = ModelFitter.run_for_fips(fips)
+        return model_fitter
+    return None
```

Comment:
Would be good to raise an alert when this happens, I'd like for this failure to not be silent

Reply:
adding a warning